medicines/web/next.config.js

```diff
@@ -1,5 +1,12 @@
 require('dotenv').config();
 
+const isMasterEnvironment =
```

Comment:
I'm really not a fan of code being aware of CI/CD stuff, can we pass in BASE_URL from the Azure build process and keep the code agnostic to implementation details?

Reply:
that wouldn't take care of the need for `/index.html` for links to static pages to work in branch environments, so we'd have to bring that in too